eox_tenant/settings/common.py

```diff
@@ -52,6 +48,7 @@ def plugin_settings(settings):
     settings.CHANGE_DOMAIN_DEFAULT_SITE_NAME = "stage.edunext.co"
     settings.EOX_TENANT_LOAD_PERMISSIONS = True
     settings.EOX_TENANT_APPEND_LMS_MIDDLEWARE_CLASSES = False
+    settings.USE_EOX_TENANT = True
```

Comment:
Why do we need this change? 

This middleware https://github.com/eduNEXT/eox-tenant/blob/be8fe48a08ba1cfd836347e9daf78e6b7d3e3d2b/eox_tenant/middleware.py#L106 is now broken because TenantSiteConfigProxy.enabled  always return True.

Probably this change is OK, but then we would need to change that middleware

Reply:
This new setting is not the problem, this is only for active or deactivate tenants, replace the setting USE_MICROSITES, the problem is that the middleware was designed to work when site_configuration is not enabled, and this pr is designed  to start to work with site_configuration, I'll check how to fix it

Reply:
I don't think that this should be set as a default. IIRC this key USE_EOX_TENANT must be set via ansible to turn the thing on